src/Stryker.Core/Stryker.Core.UnitTest/Reporters/ConsoleReportReporterTests.cs

```diff
@@ -147,7 +147,7 @@ public void ConsoleReportReporter_ShouldPrintRedOn60PercentAndLower()
                 Type = MutatorType.Arithmetic
             };
 
-            var target = new ConsoleReportReporter(new StrykerOptions("", "ReportOnly", "", 1000, null, "info", false, false, 1, 80, 60, 0), chalkMock.Object);
+            var target = new ConsoleReportReporter(new StrykerOptions(thresholdLow:60, thresholdBreak:0), chalkMock.Object);
```

Comment:
I suggest not passing any values to strykeroptions here as 60 and 0 are the default values for thresholdLow and thresholdBreak. Instead change the test name to `ConsoleReportReporter_ShouldPrintRedUnderThresholdBreak`